src/Creator/MassInvoicesCreator.php

```diff
@@ -19,16 +19,10 @@
 
 final class MassInvoicesCreator implements MassInvoicesCreatorInterface
 {
-    private InvoiceCreatorInterface $invoiceCreator;
-
-    private DateTimeProvider $dateTimeProvider;
-
     public function __construct(
-        InvoiceCreatorInterface $invoiceCreator,
-        DateTimeProvider $dateTimeProvider
+        private readonly InvoiceCreatorInterface $invoiceCreator,
+        private readonly DateTimeProvider $dateTimeProvider
```

Comment:
```suggestion
        private readonly DateTimeProvider $dateTimeProvider,
```
and in some other places